packages/ebayui-core-react/src/ebay-file-preview-card-group/file-preview-card-group.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, FC, ComponentProps } from 'react'
+import {
+    EbayFilePreviewCard,
+    EbayFilePreviewCardProps
+} from '../ebay-file-preview-card'
+import {
+    FilePreviewCardActionHandler,
+    FilePreviewCardMenuActionHandler
+} from './types'
+
+export type EbayFilePreviewCardGroupProps = ComponentProps<'div'> & {
+    cards?: EbayFilePreviewCardProps[]
+    a11yCancelUploadText?: EbayFilePreviewCardProps['a11yCancelUploadText']
+    deleteText?: EbayFilePreviewCardProps['deleteText']
+    menuActions?: EbayFilePreviewCardProps['menuActions']
+    a11ySeeMoreText?: EbayFilePreviewCardProps['a11ySeeMoreText']
+    onMenuAction?: FilePreviewCardMenuActionHandler
+    onDelete?: FilePreviewCardActionHandler
+    onCancel?: FilePreviewCardActionHandler
+}
+
+const SHOW_AMOUNT = 15 // default number of cards to show taken from marko's implementation
+
+const EbayFilePreviewGroup: FC<EbayFilePreviewCardGroupProps> = ({
+    cards = [],
+    a11yCancelUploadText,
+    deleteText,
+    menuActions,
+    a11ySeeMoreText,
+    onMenuAction,
+    onDelete,
+    onCancel,
+    ...rest
+}) => {
+    const [cardsShowing, setCardsShowing] = useState(() => SHOW_AMOUNT)
```

Comment:
No need to use the useState with a callback. just `useState(SHOW_AMOUNT)` is fine. The callback is useful when some slow code needs to be used like: `useState(() => verySlowFunction())`